dashboard/src/services/encryption.js

```diff
@@ -230,6 +230,7 @@ export const decryptItem = async (item, { decryptDeleted = false, type = "" } =
       }
     );
     capture(`ERROR DECRYPTING ${type || "ITEM"} ${item?._id} : ${errorDecrypt}`, {
+      fingerprint: [`{{ default }}`, item._id],
```

Comment:
VOir ma remarque dans l'autre PR, je pense qu'on peut garder le titre pas trompeur.
Et *peut-être* utiliser la date (updatedAt) de l'item avec l'orga (mais je ne pense pas à la réflexion)

```suggestion
    capture(`ERROR DECRYPTING ${type || "ITEM"}: ${errorDecrypt}`, {
      fingerprint: [`{{ default }}`, item?._id],
```